certbot/cli.py

```diff
@@ -366,33 +367,40 @@ def _get_help_string(self, action):
     }),
     ("register", {
         "short": "Register for account with Let's Encrypt / other ACME server",
-        "opts": "Options for account registration & modification"
+        "opts": "Options for account registration & modification",
+        "usage": "\n\n  certbot register --email user@example.com [options]"
     }),
     ("unregister", {
         "short": "Irrevocably deactivate your account",
-        "opts": "Options for account deactivation."
+        "opts": "Options for account deactivation.",
+        "usage": "\n\n  certbot unregister --email user@example.com [options]"
     }),
     ("install", {
         "short": "Install an arbitrary cert in a server",
-        "opts": "Options for modifying how a cert is deployed"
+        "opts": "Options for modifying how a cert is deployed",
+        "usage": "\n\n  certbot install --cert-path /path/to/fullchain.pem [options]"
     }),
     ("config_changes", {
         "short": "Show changes that Certbot has made to server configurations",
-        "opts": "Options for controlling which changes are displayed"
+        "opts": "Options for controlling which changes are displayed",
+        "usage": "\n\n  certbot config_changes --num NUM [options]"
     }),
     ("rollback", {
         "short": "Roll back server conf changes made during cert installation",
-        "opts": "Options for rolling back server configuration changes"
+        "opts": "Options for rolling back server configuration changes",
+        "usage": "\n\n  certbot rollback [options]"
```

Comment:
I think it's a good idea to include `--checkpoints` here. This flag allows the user to specify the number of checkpoints Certbot should rollback instead of just simply rolling back a single checkpoint.